packages/grafana-ui/src/components/Table/utils.ts

```diff
@@ -58,6 +58,7 @@ export function getColumns(data: DataFrame, availableWidth: number, columnMinWid
       accessor: (row: any, i: number) => {
         return field.values.get(i);
       },
+      sortType: 'basic',
```

Comment:
should this property not depend on field type? (so be different for number or string fields)

Reply:
Something like this? I'm not sure what sorting to use for the other types (boolean, trace...)

```ts
   const selectSortType = (type: FieldType): string => {
      switch (type) {
        case FieldType.number: return 'basic';
        case FieldType.time : return 'datetime';
        default: return 'alphanumeric';
      }
    }

    const Cell = getCellComponent(fieldTableOptions.displayMode, field);
    columns.push({
      Cell,
      id: fieldIndex.toString(),
      Header: getFieldDisplayName(field, data),
      accessor: (row: any, i: number) => {
        return field.values.get(i);
      },
      sortType: selectSortType(field.type),
      width: fieldTableOptions.width,
      minWidth: 50,
    });
```

Reply:
yes! think time should also be numeric (as time fields are mostly number ms epochs)